etc/kapacitor/kapacitor.conf

```
@@ -573,27 +573,46 @@ default-retention-policy = ""
   # Use SSL but skip chain & host verification
   insecure-skip-verify = false
   ## Optional SASL Config
-  # sasl_username = "kafka"
-  # sasl_password = "secret"
+  # sasl-username = "kafka"
+  # sasl-password = "secret"
+  ## Arbitrary key value string pairs to pass as a TOML table. For example:
+  # {logicalCluster = "cluster-042", poolId = "pool-027"}
+  # sasl-extensions = {}
   ## Optional SASL:
   ## one of: OAUTHBEARER, PLAIN, SCRAM-SHA-256, SCRAM-SHA-512, GSSAPI
   ## (defaults to PLAIN)
-  # sasl_mechanism = ""
+  # sasl-mechanism = ""
   ## used if sasl_mechanism is GSSAPI
-  # sasl_gssapi_service_name = ""
+  # sasl-gssapi-service-name = ""
   # ## One of: KRB5_USER_AUTH and KRB5_KEYTAB_AUTH
-  # sasl_gssapi_auth_type = "KRB5_USER_AUTH"
-  # sasl_gssapi_kerberos_config_path = "/"
-  # sasl_gssapi_realm = "realm"
-  # sasl_gssapi_key_tab_path = ""
-  # sasl_gssapi_disable_pafxfast = false
-  ## Access token used if sasl_mechanism is OAUTHBEARER
-  # sasl_access_token = ""
-  ## Arbitrary key value string pairs to pass as a TOML table. For example:
-  # {logicalCluster = "cluster-042", poolId = "pool-027"}
-  # sasl_extensions = {}
+  # sasl-gssapi-auth-type = "KRB5_USER_AUTH"
+  # sasl-gssapi-kerberos-config-path = "/"
+  # sasl-gssapi-realm = "realm"
+  # sasl-gssapi-key-tab-path = ""
+  # sasl-gssapi-disable-pafxfast = false
+  ## Options if sasl-mechanism is OAUTHBEARER
+  ## The service name to use when authenticating with SASL/OAUTH.
+  # ## One of: "" or custom, auth0, azuread
+  # sasl-oauth-service = ""
+  ## The client ID to use when authenticating with SASL/OAUTH.
+  # sasl-oauth-client-id = ""
+  ## The client secret to use when authenticating with SASL/OAUTH.
+  # sasl-oauth-client-secret = ""
+  ## The token URL to use when sasl-oauth-service is custom or auth0. Leave empty otherwise.
+  # sasl-oauth-token-url = ""
+  ## The expiry margin for the token.
```

A small suggestion to clarify the meaning of this property:

```suggestion
  ## The margin for the token's expiration time.
```

---

Thanks, fixed